csbuild/_testing/testcase.py

```diff
@@ -85,11 +85,11 @@ def run(self, result=None):
 		unittest.TestCase.run(self, result)
 		if self.success:
 			log.Test("	  ... <&DGREEN>[</&><&GREEN>Success!</&><&DGREEN>]")
-			TestCase._currentTestCase[1] += 1
+			TestCase._currentTestCase[1] += 1 # pylint: disable=unsupported-assignment-operation
```

Comment:
I wonder why pylint thinks this is unsupported...

Reply:
Can we better type-hint `TestCase._currentTestCase` to get pylint happy instead of disabling the check?